src/coreclr/System.Private.CoreLib/src/System/Diagnostics/Debugger.cs

```diff
@@ -85,9 +88,11 @@ public static extern bool IsAttached
         private static partial void LogInternal(int level, string? category, string? message);
 
         // Checks to see if an attached debugger has logging enabled
-        //
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        public static extern bool IsLogging();
+        public static bool IsLogging() => IsLoggingHelper();
+
+        [LibraryImport(RuntimeHelpers.QCall, EntryPoint = "DebugDebugger_IsLoggingHelper")]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        private static partial bool IsLoggingHelper();
```

Comment:
```suggestion
        private static partial bool IsLoggingInternal();
```

The recommended guidelines for names are to use `Internal` for this sort of disambiguation.